### ISSUE TITLE
How To Save File As MP4?

### ISSUE THREAD

**Issue by skthanu** [Godzil/Crunchy#52]
Can You please make a feature that allows us to save the file as any format such as mp4

**Comment by Godzil** [comment]
what is the problem with MKV?

MKV allow to store both video in h264 format, and multiples stream of subtitles in one file instead of having a mp4 there and .sub .srt at the same location, so what it the idea there?

**Comment by skthanu** [comment]
@Godzil yeah, i know mkv is better than mp4 most of the time, but i want no have an mp4 file so that i can use that more convinent for my purpose.
I know ffmpeg can help me to show this problem however it take time, so it would be Great if you can make a feture that can allow people to choose the output format.

**Comment by Godzil** [comment]
Well the 
> -m, --merge        Disables merging subtitles and videos.
should work and you will end with the .mp4 file and .ass file as they are before being merge in a mkv file.
